src/operators.ts

```diff
@@ -0,0 +1,10 @@
+import { LikeInsensitive } from './types'
+
+export const likeInsensitive = {
+  // converts the input to the filter
+  parse: (value: string): LikeInsensitive => ({
+    likeInsensitive: value
+  }),
+  // converts the filter back to the input
+  format: (obj: LikeInsensitive) => obj && obj.likeInsensitive
+}
```

Comment:
Thank you for making the first operator. I noticed that parse requires `value` to be a `string`. I think this makes sense for `likeInsensitive` as only a string makes sense.

See my comment in https://github.com/BowlingX/ra-postgraphile/pull/17#discussion_r396008900

How do you imagine writing an operator that can handle different types? For example: `contains` can be used for arrays, ranges or internet addresses. Would we create something like the following?

```
export const arrayContains<T> = {
  parse: (value: T[]): Contains => ({ contains: value }),
  ...
};

export const rangeContains<T> = {
  parse: (value: T): Contains => ({ contains: value }),
  ...
};

export const inetContains = {
  parse: (value: InternetAddress): Contains => ({ contains: value }),
  ...
};
```

Or would we ignore the type altogether?

```
export const arrayContains = {
  parse: (value: any): Contains => ({ contains: value }),
  ...
};
```

Reply:
I'm not sure about the ups and downs of the 2 options. I suppose ignoring type is fine for now if that makes things easier and quicker now. I'd rather get something working and published and get some more user feedback on using filter operators.